### PR TITLE
917 refactor front interfaces

### PR DESCRIPTION
**Pull request by Thykof**
## Delivery Quality Checklist

- [ ] **Breaking Changes in API:**
  Does this PR introduce breaking changes in the API?
    - If yes, have you considered making it backward compatible?
    - If backward compatibility is not considered, set the "breaking-change" label.

- [ ] **Changelog:**
  - [ ] For bugfix PR, set the "bugfix" label
  - [ ] If this change should not appear in changelog, use "ignore-for-changelog" label

- [ ] **Version Update Handling:**
  Have you ensured that the version update by user is handled correctly?

- [ ] **PR Dependency:**
  Does this PR depend on another PR?
    - If yes, is it necessary for the dependency to be released prior to merging this one?

- [ ] **Documentation:**
  - [ ] Are any necessary changes made to user-facing documentation?
  - [ ] Confirm that API documentation is updated with any relevant changes.
  - [ ] Check that README and other documentation files are accurate and current.

### CODE REVIEW

**Review by thomas-senechal — APPROVED** [review]

File: web/massastation/src/main.tsx [hunk 1]
```diff
@@ -56,8 +57,10 @@ const router = createBrowserRouter(
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
-    <QueryClientProvider client={queryClient}>
-      <RouterProvider router={router} fallbackElement={<Error />} />
-    </QueryClientProvider>
+    <IconContext.Provider value={{ size: '32' }}>
```
Comment (pivilartisant) [inline web/massastation/src/main.tsx:60]: quick question: why did you add back the icon context provider ?

Comment (Thykof) [reply web/massastation/src/main.tsx:60]: because the size was too small when I use npm link, so this way we explicitly set the size to 32 of all icons.
